src/kicker/main.c

```diff
@@ -184,7 +187,8 @@ void init() {
 
     // OCR0A is max val of timer before reset
     // we need 1000 clocks at 1 Mhz to get 1 millisecond
-    // if we prescale by 8, then we need 125 on timer to get 1 ms exactly
+    // ~80 us - we think - Will
+    // don't worry about it - Jeremy
     OCR0A = TIMING_CONSTANT;  // reset every millisecond
```

Comment:
The comment on this line is wrong right?

Reply:
yea, it's 80us as indicated by ln 190